File: dexguard-sample/src/main/java/ivonhoe/gradle/dexguard/MainActivity.java
package ivonhoe.gradle.dexguard;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.util.Log;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        HelloWorld helloWorld = new HelloWorld();
        helloWorld.testDexGuard();
    }

    private class HelloWorld {

        private void testDexGuard() {
            Log.d("HelloWorld", "test dexguard");
        }
    }
}

File: mine-example/src/main/java/ivonhoe/android/min/example/MainActivity.kt
package ivonhoe.android.min.example

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import ivonhoe.andrid.mine.trace.LongTimeTrace

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
    }


    fun testMethod() {
        var startTime = LongTimeTrace.methodStart()

        // do test
        Thread.sleep(200);

        LongTimeTrace.methodEnd(true, startTime);
    }
}